jba/src/visualization/common.py

```diff
@@ -253,3 +253,18 @@ def plot_tests_timeline(tests_timeline: pd.DataFrame, duplicate_attempts: List[i
         ax.get_xticklabels()[attempt - 1].set_color(DUPLICATE_COLOR)
 
     st.pyplot(fig)
+
+
+def get_edu_name_columns(df: pd.DataFrame) -> List[str]:
```

Comment:
looks strange... you can get all columns from df and intersect with columns that should be for example. This approach is more flexible and less time consuming, since now you use search among all columns 3 times for each option